docs/sources/enterprise/http_api.md

```diff
@@ -0,0 +1,72 @@
++++
+title = "Enterprise HTTP APIs"
+description = "Grafana Enterprise APIs"
+keywords = ["grafana", "enterprise", "api"]
+aliases = ["/docs/grafana/latest/enterprise/http_api/"]
+type = "docs"
+[menu.docs]
+name = "HTTP APIs"
+parent = "enterprise"
+weight = 1000
++++
+
+# Enterprise HTTP APIs
+
+## Authentication
+
+To access the API you need to authenticate yourself. For more information about API authentication, refer to [Authentication API]({{< relref "../http_api/auth.md" >}}).
+
+## Reporting API
+
+### Send report
+
+> Only available in Grafana Enterprise v7.0+.
+
+`POST /api/reports/email`
+
+Generate and send a report. This API waits for the report to be generated before returning. You are recommended to set the client's timeout to 60 seconds or more.
+
+This API endpoint is experimental and may be deprecated in a future release. On deprecation, a migration strategy will be provided and the endpoint will remain functional until the next major release of Grafana.
+
+**Example request:**
+
+```http
+POST /api/reports/email HTTP/1.1
+Accept: application/json
+Content-Type: application/json
```

Comment:
```suggestion
Content-type: application/json
```

Reply:
The name of the header is `Content-Type`, see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type